decidim-budgets/lib/decidim/api/budget_type.rb

```diff
@@ -8,9 +8,9 @@ class BudgetType < Decidim::Api::Types::BaseObject
 
       description "A budget"
 
+      field :description, Decidim::Core::TranslatedFieldType, "The description for this budget", null: false
       field :id, GraphQL::Types::ID, "The internal ID of this budget", null: false
       field :title, Decidim::Core::TranslatedFieldType, "The title for this budget", null: false
-      field :description, Decidim::Core::TranslatedFieldType, "The description for this budget", null: false
       field :total_budget, GraphQL::Types::Int, "The total budget", null: false, camelize: false
 
```

Comment:
Projects is in another block of text intentionally?

Reply:
no, the tool doesn't remove spaces ... should be fixed now.